Sources/AWSSDKSwiftCore/Credential/StaticCredential+Shared.swift

```diff
@@ -45,6 +45,7 @@ extension StaticCredential {
         on eventLoop: EventLoop) -> EventLoopFuture<StaticCredential>
     {
         let threadPool = NIOThreadPool(numberOfThreads: 1)
+        threadPool.start()
```

Comment:
Can we move these credential fixes into another PR? I would like them in ASAP, not when the rest of this PR goes in. Did you forget to start the threadPool in your previous PR?